Tests/ClientRuntimeTests/Retry/RetryIntegrationTests.swift

```diff
@@ -116,6 +118,39 @@ final class RetryIntegrationTests: XCTestCase {
         }
         try await next.verifyResult()
     }
+
+    // Test getEstimatedSkew utility method.
+    func test_getEstimatedSkew() {
+        let responseDateString = "Mon, 15 Jul 2024 01:24:12 GMT"
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "EEE, dd MMM yyyy HH:mm:ss z"
+        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+        dateFormatter.timeZone = TimeZone(abbreviation: "GMT")
+        let responseDate: Date = dateFormatter.date(from: responseDateString)!
+
+        let responseDateStringPlusTen = "Mon, 15 Jul 2024 01:24:22 GMT"
+        let estimatedSkew = getEstimatedSkew(now: responseDate, responseDateString: responseDateStringPlusTen)
+
+        XCTAssert(estimatedSkew == 10.0)
```

Comment:
nit: Use `XCTAssertEqual` to compare two values in a test assertion

Reply:
Changed as suggested.